README.md

```diff
@@ -67,7 +67,7 @@ Hardware requirements
 ---
 * Disk space: 300 GB (235GB for Bitcoin blockchain + some room)
   * Bitcoin Core pruning is not supported at the moment because it's not supported by c-lightning. It's possible to use pruning but you need to know what you're doing.
-* RAM: 2GB of memory
+* RAM: 2GB of memory. ECC memory is better.
```

Comment:
Can you explain why ECC memory is better?

Reply:
To avoid bitflips resulting in weak private keys or nonces. And to make rowhammer (slightly) harder.

Reply:
Unfortunately we can't make a footnote for that in the README.

Reply:
Perhaps a bit too hard to swallow in the README.

Reply:
I agree. As long as it doesn't have any trade-offs users should be aware of.